### DISCUSSION TITLE
Remnant Storyline

### DISCUSSION THREAD

**Discussion by SultanRTG** [endless-sky/endless-sky#9223] [Q&A] [answered]
What are all of the missions for the remnant storyline? I got to the Remnant License, but now am stuck.

**Comment by Zitchas** [comment, marked as answer]
It's pretty hard to guess where exactly you need to go; but some ideas:
- Have you been asked to bring back some samples of human technology? If so, then bringing back a plasma canon or a catalytic ramscoop or any one of a few other things (there's six options that work, currently)
- Perhaps go visit the planets in Nenia in a gascraft?
- Doing a bunch of jobs shouldn't be strictly required, but if you're just having a streak of badluck and always missing the mission random chance, this will likely give you a chance to pick up another spaceport mission.

If you post a pilot file I might be able to provide some more specific guidance.

**Reply by SultanRTG** [reply]
I got past that particular roadblock, and got to Postverta and the pocket dimension stuff. Is there anything after that in the remnant storyline?

**Reply by Zitchas** [reply]
There are a few things past there, yes. That's pretty close to the end of the "main" missions, though. 

Keep an eye out as you travel, though. There might be something interesting to spot out in the Ember Waste; likewise various opportunities to talk with the Remnant on their various worlds and learn more about them.

**Reply by paularbear45** [reply]
@Zitchas, same thing happens with me, after receiving the Remnant License, I was asked to deliver human tech, after delivering the ramscoop, nothing else appears, I try to land in and out of planets, doing courier missions and palvaret hunting and stuff, Ive tried doing 3 playthroughs and it always end up like this, Ive once completed the storyline and just got back to endless sky again so Im sure theres still more, especially Ive not yet gotten the Capital License

**Reply by TheGiraffe3** [reply]
Did you try bringing other advanced human technology?

**Reply by Zitchas** [reply]
Have you, by any chance, helped the Deep conduct scans and investigations into the Remnant?

**Reply by paularbear45** [reply]
I actually finished the remnant story line before but right now I have around 20+ mods and the remnant storyline gets stucked after I bring the Catalytic Ramscoop to Taely, Im not an expert but I tried to read each mods descriptions, as well as the data if it somehow alter anything regarding remnant timeline for quests yet I cant seem to find any, tried uninstalling all the mods and then boom, its working, maybe someone can help pinpoint the mod that actually ruins the game? I can provide the list

**Reply by Zitchas** [reply]
Might be good to see the list.

**Reply by paularbear45** [reply]
This is my modlist, some of the mods are on the official plugin lists of endless sky, some are randoms I found on reddit recommendations and ModDB, all runs perfectly too
Adjusted Hyperdrive
automata.destruction.0percent
BetterVanillaHai-0.9.2
BetterVanillaPirates-2.1.2
blended-ships-main
Businessman Mod
Capitalis-Major-1.1
Custom Outfitting (Chop Shop)
Czatrak's Ship Pack
DAIS
Easier-Korath-Automatons-1.0.0
expert.crew.members
Extra Large Jobs
Fire-Corporation-0.2
galactic.capital.investment
Investment Bank
kor.efret.shipyard
Lost in Midnight
Mega-Freight-main
Midnight-Expansion-main
Midnight-Scrapyard-main
Mil-taj-Protectorate-1.0
more.boarding.missions
MultiTech Extra Light
navy.licenses
QOL-Outfits-main
Science Stuff
sheragi-rebirth-0.10.1
Space Shuttle Start
Tales-Of-Apoxys-DX-1.0.1

**Reply by paularbear45** [reply]
Most, if not all, are also the latest version of the plug ins that I can find

**Reply by TheGiraffe3** [reply]
Try uninstalling the Midnight plugins.

**Reply by Zitchas** [reply]
Mignight is the one that I know has some Remnant content and missions in it, so most likely to have potential interference; although they've put a lot of work into it so I'd be surprised if that was the case. That beijng said, you might have stumbled into an edge case.
